<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>鼠标模拟器</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="6" column="1">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>步长休眠时间</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="event_down_sleep_time">
        <property name="text">
         <string>100</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="target_move_x">
        <property name="text">
         <string>100</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLineEdit" name="for_cnt">
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>步长</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>事件按下休眠时间</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="target_move_y">
        <property name="text">
         <string>100</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Y</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>X</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLineEdit" name="stride">
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>循环次数</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLineEdit" name="stride_delay">
        <property name="text">
         <string>10</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>清除事件休眠时间</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="clear_event_sleep_time">
        <property name="text">
         <string>30</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>554</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>模拟事件</string>
    </property>
    <addaction name="moveCursor"/>
    <addaction name="leftClick"/>
    <addaction name="leftDoubleClick"/>
    <addaction name="leftDrag"/>
    <addaction name="rightClick"/>
    <addaction name="rightDrag"/>
    <addaction name="wheelZoomIn"/>
    <addaction name="wheelZoomOut"/>
    <addaction name="middleDrag"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="actioncamera">
   <property name="text">
    <string>选择相机</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionsetting">
   <property name="text">
    <string>配置</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionlog">
   <property name="text">
    <string>操作日志</string>
   </property>
   <property name="toolTip">
    <string>操作日志</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="moveCursor_left">
   <property name="text">
    <string>光标位置移动-left</string>
   </property>
  </action>
  <action name="leftClick">
   <property name="text">
    <string>左键单击</string>
   </property>
  </action>
  <action name="leftDoubleClick">
   <property name="text">
    <string>左键双击</string>
   </property>
  </action>
  <action name="leftDrag_left">
   <property name="text">
    <string>左键点击拖动-left</string>
   </property>
  </action>
  <action name="rightClick">
   <property name="text">
    <string>右键单击</string>
   </property>
  </action>
  <action name="rightDrag_right">
   <property name="text">
    <string>右键点击拖动-right</string>
   </property>
  </action>
  <action name="wheelZoomIn">
   <property name="text">
    <string>滚轮放大 (滚轮向上滚动)</string>
   </property>
  </action>
  <action name="wheelZoomOut">
   <property name="text">
    <string>滚轮缩小 (滚轮向下滚动)</string>
   </property>
  </action>
  <action name="middleDrag">
   <property name="text">
    <string>中键点击拖动</string>
   </property>
  </action>
  <action name="moveCursor_right">
   <property name="text">
    <string>光标位置移动-right</string>
   </property>
  </action>
  <action name="moveCursor_up">
   <property name="text">
    <string>光标位置移动-up</string>
   </property>
  </action>
  <action name="moveCursor">
   <property name="text">
    <string>光标位置移动</string>
   </property>
  </action>
  <action name="leftDrag_right">
   <property name="text">
    <string>左键点击拖动-right</string>
   </property>
  </action>
  <action name="leftDrag_up">
   <property name="text">
    <string>左键点击拖动-up</string>
   </property>
  </action>
  <action name="leftDrag">
   <property name="text">
    <string>左键点击拖动</string>
   </property>
  </action>
  <action name="rightDrag">
   <property name="text">
    <string>右键点击拖动</string>
   </property>
  </action>
  <action name="rightDrag_up">
   <property name="text">
    <string>右键点击拖动-up</string>
   </property>
  </action>
  <action name="rightDrag_down">
   <property name="text">
    <string>右键点击拖动-down</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
